CLI_QUICKSTART.markdown

```diff
@@ -344,4 +344,21 @@ with a pkill command:
 $ pkill vc-server
 $ pkill vc-pas
 ```
+##  IP addresses and ports used by various Veracruz components
+
+|    IP      | Port |                              Usage                                                                       | Location(s) |
+|------------|------|----------------------------------------------------------------------------------------------------------|----------|
+| 127.0.0.1  | 6000 | Veracruz Server and Runtime Manager communicate over a TCP socket                                        | `veracruz-server/src/veracruz_server_linux.rs` |
+| 193.0.2.2  | 24   | Used to setup TAP network interface connected to a local NAT                                             | `veracruz-mcu-client/tap-setup.sh` <br><br> `veracruz-mcu-client/samples/audio-event-triangulation/prj.conf` <br><br> `veracruz-mcu-client/samples/shamir-secret-sharing/prj.conf` |
+| 193.0.2.1  |      | Used to setup TAP network interface connected to a local NAT                                             | same as `193.0.2.2:24` (above entry) |
+| 192.0.2.1  |      | Used to setup a SLIP based network interfaces connected to a local NAT                                   | `veracruz-mcu-client/slip-setup.sh` |
+| 127.0.0.1  | 3016 | Used in test for proxy attestion service                                                                 | `proxy-attestation-server/src/test.rs` |
+| 127.0.0.1  | 3010 | Used as veracruz server ip, also published for docker container over tcp                                 | `veracruz-mcu-client/run_mcu_demo_server.sh` <br><br> `veracruz-mcu-client/run_mcu_test_server.sh` <br><br> `veracruz-mcu-client/MCU_CLIENT_INSTRUCTIONS.markdown` <br><br> `workspaces/shared.mk` <br><br> `docker/Makefile`|
+| 127.0.0.1  | 3011 | Used as used as proxy attestation server ip in test/demo of server-side commands for veracruz mcu client | `workspaces/shared.mk` |
+| 127.0.0.1  | 3012 |  | `workspaces/shared.mk` |
+| 127.0.0.1  | 3013 |  | `workspaces/shared.mk` |
+| 127.0.0.1  | 3030 |  | `workspaces/shared.mk` |
+| 172.17.0.2 | 3010 | Used as used as proxy attestation server ip in test/demo of server-side commands for veracruz mcu client | `veracruz-mcu-client/run_mcu_test_server.sh` <br><br> `veracruz-mcu-client/run_mcu_demo_server.sh` <br><br> `veracruz-mcu-client/MCU_CLIENT_INSTRUCTIONS.markdown` |
```

Comment:
The description can be simplified to `Port used for proxy attestation server in MCU test case`